docs/dev_process.md

```diff
@@ -110,10 +110,14 @@ On the `master branch`:
 2. Update the change log found at `CHANGELOG.md` - see https://keepachangelog.com/en/1.0.0/ for guidelines
 3. Commit and tag the release:
    ```sh
+   $ git commit -m "Updates for version 0.1.14"
    $ git tag -a v0.1.14 -m "Release version v0.1.14"
    $ git push --tags
    ```
-4. Check that your working copy is clean by running `git clean -dxn -e __pycache__`.
+4. Check that your working copy is clean by running:
+   ```sh
+   $ git clean -dxn -e __pycache__
+   ```
```

Comment:
I think it would be much better if our build tool was configured to only include whitelisted files, rather than rely on people locally running a clean step during release. But that’s for another day.